index.html

```diff
@@ -284,12 +291,14 @@
         <code>"mark"</code> and <code>"measure"</code> [[USER-TIMING-2]],
         <code>"navigation"</code> [[NAVIGATION-TIMING-2]],
         <code>"resource"</code> [[RESOURCE-TIMING-2]],
+        <code>"server"</code> [[SERVER-TIMING]],
         <!-- TODO: add long task spec reference -->
         and <code>"longtask"</code>.</p>
       <p>
         The <dfn for="PerformanceEntry">startTime</dfn> attribute MUST return
         the time value of the first recorded timestamp of this
-        performance metric.
+        performance metric. If the startTime concept doesn't apply, a
+        performance metric MUST return a `startTime` of <code>0</code>.
```

Comment:
With server-timing, `startTime` has no meaning. The entry just below about `duration` specifies that the value can be 0. I was just following the pattern. :)